README.md

```diff
@@ -63,6 +63,8 @@ Documentation of the LSST Science Pipelines is at https://pipelines.lsst.io
         $ ingestImagesDecam.py /path/to/repo --filetype raw /path/to/raw/*.fits.fz
         $ ingestCalibs.py /path/to/repo/  --calib /path/to/calib/repo/ --calibType defect /path/to/calib/*fits --validity 999
```

Comment:
I am slightly confused, we still need to be able to ingest fringes and defects. Is there a different method for that?